From fcc3f6fa2989fbd2ce210c282fabf0bf3821d834 Mon Sep 17 00:00:00 2001
From: Vishal Verma <vishal.l.verma@intel.com>
Date: Wed, 15 Jun 2022 16:48:11 -0600
Subject: [PATCH 210/217] ndctl: update .gitignore

Add a few files and dirs to .gitignore:
  - cscope.* (since we already ignore 'tags')
  - release/ (created by scripts/prepare_release)
  - scripts/docsurgeon_parser.sh (parser generated by argbash)

Link: https://lore.kernel.org/r/20220615224813.523053-4-vishal.l.verma@intel.com
Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index aa0ce8e..eeb275f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,5 +1,8 @@
 build/
+release/
 rhel/ndctl.spec
 sles/ndctl.spec
 *.swp
 tags
+cscope.*
+scripts/docsurgeon_parser.sh
-- 
2.27.0

